code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -973,3 +973,87 @@ datum/status_effect/stabilized/blue/on_remove()
 				qdel(src)
 				qdel(linked_extract)
 	return ..()
+
+///////////////////////////////////////////////////////
+//////////////////DESTABILIZED EXTRACTS//////////////////
+///////////////////////////////////////////////////////
+
+/obj/screen/alert/status_effect/caged
+	name = "Caged"
+	desc = "You are trapped in a cage. Resist to escape."
+	icon_state = "slime_metal_cage"
+
+/datum/status_effect/cageTrapped
+	id = "slime_metal_cage"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1 //Will remove self when cage breaks.
+	alert_type = /obj/screen/alert/status_effect/caged
+	var/obj/structure/metalCage/cage
+
+/datum/status_effect/cageTrapped/on_apply()
+	RegisterSignal(owner, COMSIG_LIVING_RESIST, .proc/breakCage)
+	cage = new /obj/structure/metalCage(get_turf(owner))
+	owner.forceMove(cage)
+	return ..()
+
+/datum/status_effect/cageTrapped/tick()
+	if(!cage || owner.loc != cage)
+		owner.remove_status_effect(src)
+
+/datum/status_effect/cageTrapped/proc/breakCage()
+	if(do_after(owner, 450, FALSE, cage))
+		owner.remove_status_effect(src)
+
+/datum/status_effect/cageTrapped/on_remove()
+	if(cage)
+		qdel(cage)
+	UnregisterSignal(owner, COMSIG_LIVING_RESIST)
+
+/datum/status_effect/freon/destabilized
+	can_melt = FALSE
+
+/obj/screen/alert/status_effect/heavyBleeding
+	name = "Heavy Bleeding"
+	desc = "You are bleeding profusely. Only time can cure this."
+	icon_state = "slime_heavy_bleeding"
+
+/datum/status_effect/heavyBleeding
+	id = "heavy_bleeding"
+	duration = 150
+	alert_type = /obj/screen/alert/status_effect/heavyBleeding
+
+/datum/status_effect/heavyBleeding/tick()
+	if(iscarbon(owner))
+		var/mob/living/carbon/C = owner
+		if(ishuman(C))
+			var/mob/living/carbon/human/H = C
+			if(H.bleed_rate < 5)
+				H.bleed_rate = 5
+			if(H.bleedsuppress)
+				H.resume_bleeding()
+		else
+			C.bleed(5)
+
+/datum/status_effect/heavyBleeding/on_remove()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		H.bleed_rate = 0
+
+/obj/screen/alert/status_effect/cellularDestabilization
+	name = "Cellular Destabilization"
+	desc = "Your cells are slowly starting to deteriorate away! Perhaps a compressed carbon lifeform can cure it..."
+	icon_state = "slime_cell_destab"
+
+/datum/status_effect/cellularDestabilization
+	id = "cellular_destabilization"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	alert_type = /obj/screen/alert/status_effect/cellularDestabilization
+
+/datum/status_effect/cellularDestabilization/tick()
+	if(istype(owner.get_active_held_item(), /obj/item/reagent_containers/food/snacks/monkeycube))
+		qdel(owner.get_active_held_item())
+		to_chat(owner, "<span class='notice'>The monkey cube is absorbed into your body, bringing the cellular destabilization to a halt.</span>")
+		qdel(src)
+	else	
+		owner.adjustCloneLoss(0.5)
```

Comment:
Really dislike the design on this one. This IS certain death unless you have this one specific item that only two jobs have easy access to.

And one of those jobs is the guy that's going to be the guy throwing these around.

Reply:
In defense of my design here, botany and cargo can both obtain monkey cubes, and cryo can rapidly outheal the damage this does, so it's treat-able - though, not curable - until either source can be used.

Of course, if you feel this is still too significant to have, @TheDreamweaver and I can rework the idea! Especially since this all was designed before the recent Medical changes.

Reply:
What if the affected creature doesn't have hands/can't hold stuff? Can megafauna/most simplemobs take clone damage?

Reply:
Wow, megafauna can take clone damage too. Just yeet a slime core at one and teleport away, then wait around half an hour (33 minutes and 20 seconds for, say, the ash drake) and come back to snatch the loot (but actually don't do that because the shuttle got called 20 minutes ago).

Reply:
Idea here: Make the rainbow extract's effect be the same as a bolt of possession (or a sapience potion if the hit mob has no client/mind). If it doesn't hit a creature, have it make a lightgeist or something and try to put a ghost in that.

Reply:
I'm going to change it so it only applies to carbons. As for worries of balance, you must remember that by the time xenobio has rainbows, there are already plenty of other, easier ways of permanently ruining someones round.

Reply:
Wait, the status effect already only gets applied if the target was human, so I'll consider this resolved